smartmin/views.py

```diff
@@ -141,10 +141,6 @@ def lookup_obj_attribute(self, obj, field):
         # next up is the object itself
         obj_field = getattr(obj, curr_field, None)
 
-        # if it is callable, do so
-        if obj_field and getattr(obj_field, '__call__', None):
```

Comment:
There's a chance someone is relying on this behaviour but I think it's better if views explicitly call object methods than doing it automatically here - as this doesn't take into account callables with parameters, or callables that we don't actually want to call.